devonte-duncan/exercises/answers.js

```diff
@@ -0,0 +1,52 @@
+// Question 1: Looping a Triange
+function triangle() {
+  // write your answer to question one below
+  let hash = "";
+  let counter = 0;
+  
+  while(counter <= 6){
+    let i;
+    for (i = 0; i < 7; i++){
+      hash = hash + "#";
+      console.log(hash);
+      ++counter;
+    }
+  }
+
+}
+
+// Question 2: FizzBuzz
+function fizzBuzz() {
+  // write your answer to question two below
+  let counter = 0;
+  
+  while(counter <= 99){
+    counter++
+    if(counter % 3 === 0){
+      console.log("Fizz");
+    }else if(counter % 5 === 0 && counter % 3 !== 0){
+      console.log("Buzz");
+    }else if(counter % 5 === 0 && counter % 3 === 0){
+      console.log("FizzBuzz");
+    }else{
+      console.log(counter);
+    }
+  }
+}
+
+// Question 3: Chess Board
+function chessBoard() {
+  // write your answer to question three below
+ let rowOne = " # # # #";
```

Comment:
Haha clever. My intention was for you to build each row one hashtag at a time but I can't fault you for working smarter not harder 😉